app/main/appium.js

```diff
@@ -57,6 +57,9 @@ function connectStartServer (win) {
     // than the typical behavior, which is process.exit o_O
     args.throwInsteadOfExit = true;
 
+    // Let Appium Desktop automatically download the matching ChromeDriver
+    args.allowInsecure.push('chromedriver_autodownload');
```

Comment:
Here should be configurable in the advanced tab.
Could you remove this for now or include this capability in the advanced tab?
<img src="https://user-images.githubusercontent.com/5511591/86267549-d6071900-bc01-11ea-987e-5b655deb4efd.png" width=300>

Because some users would not download binaries automatically under a slow network, for example.
Then, this flag should be able to be disabled.
(I saw such comments)

Reply:
Good remark, on the other hand, this is something that would definitely break the usage for Android if the correct CrhomeDriver is not downloaded. I also wonder how many people even look into these settings.
Can we enable it by default and let people disable it if needed?

Reply:
Yes, it seems good > turn it on by default
My main point is adding the allowInsecure in the advanced tab instead of hardcoding it (in this PR or as a separate PR)

Reply:
@KazuCocoa 

I just looked into it again, but AD already has `allowInsecure` enabled, that's why I pushed it (otherwise the push would fail). This is a screenshot of Appium 1.17. So is this a bug in the current version?

Last but not least, this is a 1 time thing IF they start with Chrome browser OR use a hybrid app where they switch to the webview. Should we still make it configurable (makes the code more complex with error handling and all kinds of other situations) or keep it this way. It's max 15MB that will be downloaded for a specific local device with Chrome

![image](https://user-images.githubusercontent.com/11979740/86435950-7a23b980-bd01-11ea-9fe9-05b9fcd557c0.png)

And it's disabled here 
![image](https://user-images.githubusercontent.com/11979740/86435983-9293d400-bd01-11ea-9b69-92152a4926b2.png)

Reply:
I see. The default value is an empty array. When we specify arg like `--allow-insecure=foo,bar` for appium server in the launching time, it will get the body.
So, in the advanced tab, the option may be text field like server address, log level etc instead of a simple checkbox.

Reply:
Ahh, ok. Then I'll add a field here to automatically download ChromeDriver and enable it by default.

Do you have any other feedback on this PR that I need to fix?

Reply:
Fixed it with this based on @mykola-mokhnach his feedback

![image](https://user-images.githubusercontent.com/11979740/86476379-b4637a00-bd46-11ea-9b34-91eda5897ffd.png)

Reply:
👍 
It seems reasonable to me